src/styles/components/_home.scss

```
@@ -5,9 +5,12 @@ $breakpoint-tablet: 820px;
   grid-template-columns: 1fr;
   gap: 1rem;
   width: 100%;
+  background-color: var(--background-color);
+  color: var(--font-color);
 
   &-top__top-image {
     width: 100%;
+    background-color: var(--background-color);
   }
 
```

If what you're saying is true, that Dustin doesn't want this image to appear in dark mode, then you can just remove it like this.
```suggestion
  @media (prefers-color-scheme: dark) {
    &-top__top-image {
      display: none;
    }
  }
```

---

Actually, what he meant is that styling the image to appear is outside the scope of this card. If it appears blacked-out on this card, that is fine.